Engine/src/Engine/ResourceManagement/ShaderGenerator.h

```diff
@@ -11,14 +18,28 @@ namespace Engine
 	{
 	public:
 		ShaderGenerator(const char* vertexSource, const char* fragmentSource);
-		~ShaderGenerator();
 		int getProgramId() const;
 
 	private:
 		int m_programId;
 		int genShader(const char* source, GLenum shaderType);
 		bool requestStatus(GLuint shaderObj, GLenum status);
 	};
+
+	class ShaderNorms {
+	public:
+		ShaderNorms();
+		~ShaderNorms();
+		void addShader(string);
```

Comment:
this isn't in the style guidelines, but I believe we've been consisting with parameter names in function signatures as well as definitions. Lets keep that consistent

Reply:
Done